src/routines/get_results.jl

```diff
@@ -16,6 +16,24 @@ function _result_dataframe(variable::JuMP.Containers.DenseAxisArray)
 
 end
 
+function _result_dataframe_d(constraint::JuMP.Containers.DenseAxisArray)
```

Comment:
We should add to this function a try - catch when JuMP throws an error because it can't find the dual.

Reply:
Do you want it to be like this ??
```julia
try result[ix] = JuMP.dual(constraint[name])
catch
    result[ix] = NAN
end
```